tools/protobuf-compiler/src/main.rs

```diff
@@ -217,3 +216,25 @@ fn get_buf_from_env() -> PathBuf {
         .or_else(|| which::which("buf").ok())
         .expect(&msg)
 }
+
+fn purge_out_dir(path: impl AsRef<Path>) {
+    for entry in read_dir(path)
+        .expect("should be able to read target folder for generated files")
```

Comment:
Would be helpful to use `path.display()` instead of "target folder" or "out dir" in the `expect` calls here and below.

Reply:
Done in [6d0092d](https://github.com/astriaorg/astria/pull/1825/commits/6d0092d8130dea8717f5f924b729ea25ef8e634c)